client/my-sites/exporter/exporter.jsx

```diff
@@ -52,8 +53,16 @@ export default React.createClass( {
 		} = this.props;
 		const siteId = this.props.site.ID;
 
-		const exportAll = () => startExport( siteId );
-		const exportSelectedItems = () => startExport( siteId, { exportAll: false } );
+		const exportAll = () => {
+			analytics.tracks.recordEvent( 'calypso_export_start_click', { scope: 'all' } );
```

Comment:
I ended up moving the track event to the connector component in 293b53e4f5a4e7faf9aa6590be51207cf32b851b - that way we can avoid duplicating the event without inserting it into the action creator. Now that I read this again, that might be what you were suggesting all along? :)